projects/observability/src/shared/components/explore-query-editor/order-by/explore-query-order-by-editor.component.ts

```diff
@@ -98,6 +100,10 @@ export class ExploreQueryOrderByEditorComponent implements OnChanges {
   private readonly defaultDirection: GraphQlSortDirection = SortDirection.Asc;
   private readonly incomingOrderByExpressionSubject: Subject<ExploreOrderBy> = new ReplaySubject(1);
   private readonly contextSubject: Subject<TraceType> = new ReplaySubject(1);
+  private readonly notSupportedAggregations: MetricAggregationType[] = [
```

Comment:
are these not allowed for all attributes?

Reply:
Yes, you're right. Renamed